### PR TITLE
next: rollout 32.20200505.1.0

### PR DESCRIPTION
**Pull request by sohankunkerkar**

### CODE REVIEW

**Comment by jlebon** [comment]
Sorry, saw this a bit late and it's past the starting rollout. Maybe it's better anyway if we schedule them to start when some folks are online. Let's say 07:00 UTC? That way @lucab at least should be online. :)

**Comment by dustymabe** [comment]
Another option is 10AM eastern which will give us a chance to review/merge these in the morning in case we don't get to it tonight. 

```
$ date -d '2020/05/07 10:00EDT' +%s
1588860000
```

**Comment by dustymabe** [comment]
Can you fix the commit message to not include "Please enter the commit message for your changes. Lines starting"

File: streams/next.json [hunk 1]
```diff
@@ -156,77 +156,81 @@
             "images": {
                 "aws": {
                     "regions": {
+                        "af-south-1": {
+                            "release": "32.20200505.1.0",
+                            "image": "ami-07ae04381c98fe76e"
+                        },
```
Comment (dustymabe) [inline streams/next.json:162]: look at that.. we automatically added a new region. Nice

File: updates/next.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "stream": "next",
   "metadata": {
-    "last-modified": "2020-04-17T18:59:10Z"
+    "last-modified": "2020-05-07T00:20:07Z"
```
Comment (dustymabe) [inline updates/next.json:4]: tip: I usually just copy/paste the `last-modified` timestamp from the other file (the one generated with `fedora-coreos-stream-generator`)

this is completely optional, though

**Comment by sohankunkerkar** [comment]
> Can you fix the commit message to not include "Please enter the commit message for your changes. Lines starting"

Fixed

**Comment by dustymabe** [comment]
```
$ date -d @1588860000
Thu 07 May 2020 10:00:00 AM EDT
```

LGTM

**Pull request merged**